client/src/utils/toast.js

```diff
@@ -3,15 +3,19 @@ import { toast } from 'react-toastify'
 import '../styles/toast.scss'
 
 export const showToast = (message, type) => {
+
   const options = {
     theme: 'colored',
     position: 'bottom-right',
     autoClose: 2000,
     hideProgressBar: true,
   }
-  return () => {
-    type === 'success' && toast.success(message, { ...options })
-    type === 'warn' && toast.warn(message, { ...options })
-    type === 'error' && toast.error(message, { ...options })
-  }
+
+  return type
+    ? type === 'success' 
+      ? toast.success(message, {...options})
+      : type === 'warn'
```

Comment:
lets not nest this, i'd prefer to keep it to always have a type specified to you should modify your calls accordingly